endpoints/integration_test.go

```diff
@@ -325,7 +271,154 @@ func TestReadinessCheck(t *testing.T) {
 	}
 }
 
-func TestMultiPutRequestGotStored(t *testing.T) {
+func TestNegativeTTL(t *testing.T) {
+	// Input
+	inReqBody := fmt.Sprintf("{\"puts\":[{\"type\":\"json\",\"value\":\"<tag>YourXMLcontentgoeshere.</tag>\",\"ttlseconds\":-1}]}")
+	inRequest, err := http.NewRequest("POST", "/cache", strings.NewReader(inReqBody))
+	assert.NoError(t, err, "Failed to create a POST request: %v", err)
+
+	// Expected Values
+	expectedErrorMsg := "Error request ttlseconds value must not be negative.\n"
+	expectedStatusCode := http.StatusBadRequest
+
+	// Set up server to run our test
+	testRouter := httprouter.New()
+	testBackend := backends.NewMemoryBackend()
+
+	testRouter.POST("/cache", NewPutHandler(testBackend, 10, true))
+	testRouter.GET("/cache", NewGetHandler(testBackend, true))
```

Comment:
I think you can delete this line since the post is all you need.

Reply:
Good catch. Done